src/example-types/metricGroups/pivot.js

```diff
@@ -353,6 +355,14 @@ let createPivotScope = (node, schema, getStats) => {
         'columns'
       )
 
+      //Add any props that should be hoisted for this col agg and remove from agg to be hoisted
+      hoistProps = _.merge(
+        _.get('hoistProps', columnsStatsAggs),
+        hoistProps,
+        {}
+      )
```

Comment:
non blocking style comment:  noticed that hoistProps for col agg was spaced out on separate lines while hostProps for row agg was all in one line.  I would assume we would want to maintain similar or same style either collapsed in one line or spaced out one item per physical line.  Thanks, non blocking comment.

Reply:
Generally I use the character break of 80 to determine this and not code patterns if that is what you are asking. There is no Code Style document that we curate though from my understanding, and it might be something worth creating.